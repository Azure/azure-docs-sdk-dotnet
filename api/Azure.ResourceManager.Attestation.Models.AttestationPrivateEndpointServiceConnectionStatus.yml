### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus&text=AttestationPrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.op_Equality(Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus,Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.op_Inequality(Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus,Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Equals(Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet-preview
uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus
commentId: T:Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus
namespace: Azure.ResourceManager.Attestation.Models
name: AttestationPrivateEndpointServiceConnectionStatus
fullName: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus
nameWithType: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.Attestation.dll
packagesWithMoniker:
- value: Azure.ResourceManager.Attestation v1.0.0-beta.3
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct AttestationPrivateEndpointServiceConnectionStatus : IEquatable<Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type AttestationPrivateEndpointServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure AttestationPrivateEndpointServiceConnectionStatus

      Implements IEquatable(Of AttestationPrivateEndpointServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet-preview
summary: The private endpoint connection status.
metadata:
  api_name:
  - Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.Attestation.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus
  - Azure::ResourceManager::Attestation::Models::AttestationPrivateEndpointServiceConnectionStatus
  - AttestationPrivateEndpointServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet-preview
