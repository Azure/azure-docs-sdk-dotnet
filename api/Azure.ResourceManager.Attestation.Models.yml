### YamlMime:NetNamespace
classes:
- uid: Azure.ResourceManager.Attestation.Models.ArmAttestationModelFactory
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateLinkResource
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateLinkServiceConnectionState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationProviderCreateOrUpdateContent
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationProviderPatch
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationServiceCreationSpecificParams
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.JsonWebKey
  monikers:
  - azure-dotnet-preview
structs:
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointConnectionProvisioningState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.AttestationServiceStatus
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Attestation.Models.PublicNetworkAccessType
  monikers:
  - azure-dotnet-preview
uid: Azure.ResourceManager.Attestation.Models
commentId: N:Azure.ResourceManager.Attestation.Models
name: Azure.ResourceManager.Attestation.Models
fullName: Azure.ResourceManager.Attestation.Models
devLangs:
- csharp
monikers:
- azure-dotnet-preview
metadata:
  f1_keywords:
  - Azure.ResourceManager.Attestation.Models
  - Azure::ResourceManager::Attestation::Models
  - Models
  - Attestation.Models
  - Attestation::Models
  helpviewer_keywords:
  - Azure.ResourceManager.Attestation.Models namespace [.NET]
  monikers:
  - azure-dotnet-preview
