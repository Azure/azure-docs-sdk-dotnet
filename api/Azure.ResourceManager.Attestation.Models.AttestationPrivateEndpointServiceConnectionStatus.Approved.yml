### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus&text=AttestationPrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Approved
  nameWithType: AttestationPrivateEndpointServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As AttestationPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.Attestation.Models
name: Approved
fullName: Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Approved
nameWithType: AttestationPrivateEndpointServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.Attestation.dll
packagesWithMoniker:
- value: Azure.ResourceManager.Attestation v1.0.0-beta.3
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Approved
  - Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.Attestation.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.Attestation.Models.AttestationPrivateEndpointServiceConnectionStatus.Approved
  - Azure::ResourceManager::Attestation::Models::AttestationPrivateEndpointServiceConnectionStatus::Approved
  - Approved
  - AttestationPrivateEndpointServiceConnectionStatus.Approved
  - AttestationPrivateEndpointServiceConnectionStatus::Approved
  helpviewer_keywords:
  - AttestationPrivateEndpointServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct AttestationPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
